README.md

```diff
@@ -97,6 +97,17 @@ extension Data {
 }
 ```
 
+- 5) Add the following to your targets info.plist (required for iOS 14 and above)
+
+```xml
+<key>NSBonjourServices</key>
+<array>
+	<string>_pusher._tcp</string>
```

Comment:
Can you also add `<string>_pusher._udp</string>` too?
So we can have an exhaustive example.

Reply:
Done :)

Reply:
Thank you very much for this PR!